src/sentry/api/bases/organization_events.py

```diff
@@ -424,9 +424,15 @@ def handle_readable_device(
                 if readable_value:
                     result["readable"] = readable_value
 
+    def set_granularity(
```

Comment:
do we need this `set_granularity` function? Could we just set granularity in the two locations this is called instead?

Reply:
Yeah we could, i can change that